microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -3050,6 +3050,234 @@ Receiver:
+   * [ICoreWebView2Notification::get_ShouldRenotify](/microsoft-edge/webview2/reference/win32/icorewebview2notification#get_shouldrenotify)
+   * [ICoreWebView2Notification::get_Tag](/microsoft-edge/webview2/reference/win32/icorewebview2notification#get_tag)
+   * [ICoreWebView2Notification::get_Timestamp](/microsoft-edge/webview2/reference/win32/icorewebview2notification#get_timestamp)
+   * [ICoreWebView2Notification::get_Title](/microsoft-edge/webview2/reference/win32/icorewebview2notification#get_title)
+   * [ICoreWebView2Notification::GetVibrationPattern](/microsoft-edge/webview2/reference/win32/icorewebview2notification#getvibrationpattern)
+   * [ICoreWebView2Notification::remove_CloseRequested](/microsoft-edge/webview2/reference/win32/icorewebview2notification#remove_closerequested)
+   * [ICoreWebView2Notification::ReportClicked](/microsoft-edge/webview2/reference/win32/icorewebview2notification#reportclicked)
+   * [ICoreWebView2Notification::ReportClosed](/microsoft-edge/webview2/reference/win32/icorewebview2notification#reportclosed)
+   * [ICoreWebView2Notification::ReportShown](/microsoft-edge/webview2/reference/win32/icorewebview2notification#reportshown)
+
+* [ICoreWebView2NotificationCloseRequestedEventHandler](/microsoft-edge/webview2/reference/win32/icorewebview2notificationcloserequestedeventhandler)<!-- Win32-only -->
+
+* [ICoreWebView2NotificationReceivedEventArgs](/microsoft-edge/webview2/reference/win32/icorewebview2notificationreceivedeventargs)
+   * [ICoreWebView2NotificationReceivedEventArgs::get_Handled](/microsoft-edge/webview2/reference/win32/icorewebview2notificationreceivedeventargs#get_handled)
+   * [ICoreWebView2NotificationReceivedEventArgs::get_Notification](/microsoft-edge/webview2/reference/win32/icorewebview2notificationreceivedeventargs#get_notification)
+   * [ICoreWebView2NotificationReceivedEventArgs::get_SenderOrigin](/microsoft-edge/webview2/reference/win32/icorewebview2notificationreceivedeventargs#get_senderorigin)
+   * [ICoreWebView2NotificationReceivedEventArgs::GetDeferral](/microsoft-edge/webview2/reference/win32/icorewebview2notificationreceivedeventargs#getdeferral)
+   * [ICoreWebView2NotificationReceivedEventArgs::put_Handled](/microsoft-edge/webview2/reference/win32/icorewebview2notificationreceivedeventargs#put_handled)
+
+* [ICoreWebView2NotificationReceivedEventHandler](/microsoft-edge/webview2/reference/win32/icorewebview2notificationreceivedeventhandler)<!-- Win32-only -->
+
+* [`COREWEBVIEW2_TEXT_DIRECTION_KIND` Enum](/microsoft-edge/webview2/reference/win32/webview2-idl#corewebview2_text_direction_kind)
+   * `COREWEBVIEW2_TEXT_DIRECTION_KIND_DEFAULT`
+   * `COREWEBVIEW2_TEXT_DIRECTION_KIND_LEFT_TO_RIGHT`
+   * `COREWEBVIEW2_TEXT_DIRECTION_KIND_RIGHT_TO_LEFT`
+
+---
+
+
+<!-- ====================================================================== -->
+## Save as (Incoming)
```

Comment:
where to put Save As APIs?

rendered incoming section:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-3253&tabs=dotnetcsharp#save-as-incoming

outline at top of PR main page:
https://github.com/MicrosoftDocs/edge-developer/pull/3253

@nishitha-burman @peiche-jessica

Reply:
making the heading more verbose and specific could be good, in case different releases add different SaveAs functionality.

such heading text is linkable as a link destination, eg
https://learn.microsoft.com/microsoft-edge/webview2/release-notes/#saveas
which is somewhat unstable; liable to change to #saveas-1 or #saveas-2 to scroll to that exact section, if a newer heading w/ same wording is added later eg when an API entry moves from Experimental in Prerelease, to Stable in Prerelease, to Stable in Release.  

But that type of ambiguity is fine, b/c normally you want the latest, topmost entry, anyway.  
The greater concern or need is, differentiate entries that add different SaveAs functionality (rather than merely moving the same new functionality through the stages pipeline).

Reply:
result (from top of PR main page - outline):
```
## Browser features
#### Printing
#### Cookies
#### Image capture
#### Downloads
#### Save as -- NEW SECTION
#### Web notification handling -- NEW SECTION
#### Permissions
#### Context menus
#### Status bar
#### Fluent overlay scrollbars
#### User Agent
#### Autofill
#### Audio
#### ...
```